jobs/index.js

```diff
@@ -28,14 +51,16 @@ function loadConsumer(listQueueName) {
     }
 }
 
-function setupJob() {
+async function setupJob() {
+    const listQueue = await getListQueue();
```

Comment:
should store listQueue as global, no need to create a new queue when call this function